routes/orders.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const ordersRouter = express.Router();
+const { requireUser, isAdmin } = require("./utils");
+const { getAllOrders, getCartByUser, createOrder } = require('../db/utils');
+
+ordersRouter.get("/", [requireUser, isAdmin], async (req, res, next) => {
+  try {
+    const orders = await getAllOrders();
```

Comment:
Set up conditional check for orders and send back data if positive, or a friendly user message if not.